src/libraries/Native/build-native.sh

```diff
@@ -49,10 +49,13 @@ source "$__RepoRootDir"/eng/native/build-commons.sh
 
 if [[ "$__BuildArch" == wasm ]]; then
     if [[ -z "$EMSDK_PATH" ]]; then
-        echo "Error: Should set EMSDK_PATH environment variable pointing to emsdk root."
+        echo "Error: You need to set the EMSDK_PATH environment variable pointing to the emscripten SDK root."
         exit 1
     fi
     source "$EMSDK_PATH"/emsdk_env.sh
+
+    export CLR_CC=$(which emcc)
+    export CLR_CXX=$(which em++)
```

Comment:
A nit, typically posix-y `command -v` is preferred over `which` in this repo. (same goes for `which false` below)

Reply:
@am11 I tried that but it didn't work since we expect the full path here and `command -v` doesn't do that (for `false` at least)